### ISSUE TITLE
Uncaught TypeError: Arguments to path.join must be strings

### ISSUE THREAD

**Issue by rahul286** [AtomLinter/Linter#332]
[Enter steps to reproduce below:]

1. I just opened Atom

**Atom Version**: 0.170.0
**System**: Mac OS X 10.10.1
**Thrown From**: [linter](https://github.com/AtomLinter/Linter) package, v0.9.1

### Stack Trace

Uncaught TypeError: Arguments to path.join must be strings

```
At path.js:390

TypeError: Arguments to path.join must be strings
  at Object.exports.join (path.js:390:15)
  at LinterJsYaml.Linter.getNodeExecutablePath (/Users/rahul286/.atom/packages/linter/lib/linter.coffee:114:10)
  at LinterJsYaml.Linter.getCmdAndArgs (/Users/rahul286/.atom/packages/linter/lib/linter.coffee:90:25)
  at LinterJsYaml.Linter.lintFile (/Users/rahul286/.atom/packages/linter/lib/linter.coffee:122:24)
  at /Users/rahul286/.atom/packages/linter/lib/linter-view.coffee:138:18
  at Array.forEach (native)
  at /Users/rahul286/.atom/packages/linter/lib/linter-view.coffee:137:18
  at Object.oncomplete (fs.js:93:15)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "autoclose-html",
      "atom-html-preview"
    ]
  },
  "linter": {}
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v1.1.0
hex, v0.3.10
jsonlint, v0.4.0
linter, v0.9.1
linter-js-yaml, v0.1.3
minimap, v3.5.4
tabs-to-spaces, v0.8.0

# Dev
No dev packages
```

/cc @atom/core

**Comment by dmnd** [comment]
Dupe of #329 which was fixed in v0.10.0. Please upgrade.

**Comment by rahul286** [comment]
Thanks for quick followup.
